packages/amazonq/.changes/next-release/Feature-4a1c951a-2289-4808-b106-86fe4f03528e.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Feature",
+	"description": "proactively show code generation iterations"
```

Comment:
One suggestion to be more specific to our vertical here and talk about the counts - "Amazon Q /dev: proactively show code generation iterations count"

Reply:
Updated.